src/emqx_mgmt_cli.erl

```diff
@@ -343,23 +343,16 @@ plugins(["unload", Name]) ->
     end;
 
 plugins(["reload", Name]) ->
-    try
-        PluginName = list_to_existing_atom(Name),
-        Config = gen_config(PluginName),
-        Plugin = emqx_plugins:find_plugin(PluginName),
-        case Plugin#plugin.active of
-            false ->
-                load_plugin(PluginName, Config);
-            true ->
-                case emqx_plugins:unload(PluginName) of
-                    ok ->
-                        load_plugin(PluginName, Config);
-                    {error, Reason} ->
-                        emqx_cli:print("Reload plugin error: ~p~n", [Reason])
-                end
-        end
-    catch _ : _Error : Stacktrace ->
-        emqx_cli:print("Reload plugin error:~p~n", [Stacktrace])
+    case catch list_to_existing_atom(Name) of
```

Comment:
Do not use case catch if you don't want to catch all the stack trace when the exception happens.